userspace/libsinsp/sinsp_filter_transformer.cpp

```diff
@@ -107,6 +108,23 @@ bool sinsp_filter_transformer::transform_type(ppm_param_type& t) const {
 		case PT_FSPATH:
 		case PT_FSRELPATH:
 			// for BASENAME, the transformed type is the same as the input type
+			return !is_list;
+		default:
+			return false;
+		}
+	}
+	case FTR_LEN: {
+		if(is_list) {
+			t = PT_UINT64;
+			flags = 0;
```

Comment:
```suggestion
			flags = flags & ~EPF_IS_LIST;
```
There may be other flags set.